src/index.html

```diff
@@ -6,7 +6,72 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="main__calendar calendar calendar--starting-day-thu calendar--month-length-30">
+        <div class="calendar__day"></div>
+
+        <div class="calendar__day"></div>
+
+        <div class="calendar__day"></div>
+
+        <div class="calendar__day"></div>
```

Comment:
i don't think it has to be empty lines between each tag in this case, but it's on your taste mb